### PR TITLE
add recipe for spenc

### PR DESCRIPTION
**Pull request by knaaptime**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by ljwolf** [comment]
I am definitely willing to maintain this package!

**Comment by knaaptime** [comment]
this is failing because the license file wasn't packaged with the last pypi release

File: recipes/spenc/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+
+test:
+  imports:
+    - spenc
+
+about:
+  home: http://github.com/ljwolf/spenc
+  license: BSD-3-Clause
+  license_family: BSD
+  license_file: license.txt
```
Comment (chrisburr) [inline recipes/spenc/meta.yaml]: You can include a copy in the recipe directory to avoid the failure.

File: recipes/spenc/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,46 @@
+
```
Comment (chrisburr) [inline recipes/spenc/meta.yaml]:
```suggestion
```

**Comment by knaaptime** [comment]
thank you!

File: recipes/spenc/meta.yaml (hunk 1 above)
Comment (chrisburr) [reply recipes/spenc/meta.yaml]: @knaaptime Can you add this file?

**Pull request merged**